src/register.js

```diff
@@ -90,7 +91,7 @@ export function wrapDidDocument(did, owner, history) {
           const type = attrTypes[match[4]] || match[4]
           const encoding = match[6]
           switch (section) {
-            case 'pub':
+            case 'pub': {
```

Comment:
Are these curly braces around the case body necessary?  If not can we not add them them extraneously and maintain consistency with the rest of the code?

Reply:
I didn't touch this part. Is from the previous code already. Should I fix it anyways?

Reply:
It is showing as a change in the diff